File: 백준/Bronze/34183. SUAPC 의자 준비하기/SUAPC 의자 준비하기.c
#include <stdio.h>
int main(void) {
    int n,m,a,b,rlt;
    scanf("%d %d %d %d",&n,&m,&a,&b);
    rlt=n*3-m;
    printf("%d",rlt>0?rlt*a+b:0);
    return 0;
}

File: 백준/Bronze/34183. SUAPC 의자 준비하기/SUAPC 의자 준비하기.java
import java.util.Scanner;
import java.util.Arrays;
public class Main {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        int[] arr=Arrays.stream(scan.nextLine().split(" "))
        .mapToInt(Integer::parseInt).toArray();
        int rlt=arr[0]*3-arr[1];
        System.out.println(rlt>0?rlt*arr[2]+arr[3]:0);
    }
}